packages/amplication-git-utils/src/types.ts

```diff
@@ -33,9 +34,66 @@ export interface GitProvidersConfiguration {
   bitBucketConfiguration: BitBucketConfiguration;
 }
 
+export interface OAuthProviderOrganizationProperties
+  extends OAuthData,
+    CurrentUser {
+  links: { avatar: LinksMetadata };
+  username: string;
+  useGroupingForRepositories: boolean;
+  uuid: string;
+  displayName: string;
+  accessToken: string;
+  refreshToken: string;
+  tokenType: string;
+  expiresAt: number;
+  scopes: string[];
+}
```

Comment:
what am I missing?
OAuthData includes these properties
accessToken: string;
  refreshToken: string;
  tokenType: string;
  expiresAt: number;
  scopes: string[];
  
  and CurrentUser includes these properties
  links: { avatar: LinksMetadata };
  username: string;
  useGroupingForRepositories: boolean;
  uuid: string;
  displayName: string;
  
  so what are we extending here?

Reply:
done! Good spot!